Add reinforcement learning example for Fabric (#16506)
Lightning-AI/lightning: examples/fabric/reinforcement_learning/README.md

# Proximal Policy Optimization - PPO implementation powered by Lightning Fabric

This is an example of a Reinforcement Learning algorithm called [Proximal Policy Optimization (PPO)](https://arxiv.org/abs/1707.06347) implemented in PyTorch and accelerated by [Lightning Fabric](https://pytorch-lightning.readthedocs.io/en/latest/fabric/fabric.html).

> The goal of Reinforcement Learning is to learn intelligent agents that are able to act in their surrounding environment maximizing the cumulative reward received by it. This can be depicted in the following figure:

Reviewer: ```suggestion
The goal of Reinforcement Learning is to train agents to act in their surrounding environment maximizing the cumulative reward received from it. This can be depicted in the following figure:
```
